pegasus/sites.v3/code.org/views/census_map.haml

```diff
@@ -53,8 +53,8 @@
 
     var layer = new google.maps.FusionTablesLayer({
       query: {
-        select: 'type',
-        from: '1G3wfhnX2LFkgkJfDA6iw0Pq3t-6pl9wjvqUdLY7B'
+        select: 'location',
```

Comment:
I had to change this to `select: 'location'` instead of `select: 'type'` to get things working with my test fusion table.
It worked fine either way with the production table. The weird thing is that the fusion table docs all say that this needs to be a location column, so it is possible that when set to type it was just ignored. Not clear why it worked for the prod table and not my test table.